modules/cats/src/main/scala/pureconfig/module/cats/package.scala

```diff
@@ -34,5 +42,10 @@ package object cats {
   implicit def nonEmptySetReader[T](implicit reader: ConfigReader[SortedSet[T]]): ConfigReader[NonEmptySet[T]] =
     ConfigReader.fromCursor(fromNonEmpty[SortedSet, NonEmptySet, T](NonEmptySet.fromSet))
   implicit def nonEmptySetWriter[T](implicit writer: ConfigWriter[SortedSet[T]]): ConfigWriter[NonEmptySet[T]] =
-    ConfigWriter.fromFunction(nel => writer.to(nel.toSortedSet))
+    ConfigWriter.fromFunction(nonEmptySet => writer.to(nonEmptySet.toSortedSet))
+
+  implicit def nonEmptyMapReader[A, B](implicit reader: ConfigReader[Map[A, B]], ord: Ordering[A]): ConfigReader[NonEmptyMap[A, B]] =
```

Comment:
@nigredo-tori there is actually a problem with `NonEmptyMap` where `cats.Order` is not actually required. But they cannot fix it now as it would break the binary compatibility. The only thing that is required is `Ordering` to build `SortedMap`. https://github.com/typelevel/cats/pull/2318

Reply:
@NicolaeNMV , I am aware that we only use `Ordering` inside. However this is probably more of a temporary (as you pointed out) implementation detail for `NonEmptyMap`, and we shouldn't leak this in our signature. See the `NonEmptyMap` implementation - it uses `Order` everywhere. We should do the same.

Reply:
@nigredo-tori users will have to import specific Order instances for their key type. Ex. `cats.instances.string._` to bring `Order[String]` in. Do you think that's a problem ?
There are less imports required if scala's `Ordering` is to be used, because the Ordering of the common types are in the scope by default.
A thing we could do to make the user's life easier is to have some default imports for String, we could bring Order[String] into the scope. WDYT ?
`package object cats extends StringInstances`

Reply:
> Do you think that's a problem ?

@NicolaeNMV, I think that importing instances is reasonable - that's how Cats work in the first place, so this behavior would not be surprising. I just do `import cats.implicits._` most of the time, anyway.

In any case, I think it's safe to assume that if someone uses `NonEmptyMap` (not the most advertised Cats type), they know to import the corresponding `Order` instance.

> A thing we could do to make the user's life easier is to have some default imports for String, we could bring Order[String] into the scope. WDYT ?

For the love of all that is holy, no! Cats goes to great length to keep importing instances explicit and predictable. The last thing we need is some `pureconfig` package silently exposing another set of instances.